xml/System.Web.Security/Roles.xml

```diff
@@ -24,7 +24,7 @@
   
  Users can belong to more than one role. For example, if your site is a discussion forum, some users might be in the role of both Members and Moderators. You might define each role to have different privileges on the site, and a user who is in both roles would then have both sets of privileges.  
   
- To enable role management for your ASP.NET application, use the  element of the `system.web` section in the Web.config file for your application, as shown in the following example.  
+ To enable role management for your ASP.NET application, use the `<roleManager>` element of the `system.web` section in the Web.config file for your application, as shown in the following example.  
```

Comment:
roleManager element: https://docs.microsoft.com/en-us/previous-versions/dotnet/netframework-4.0/ms164660(v%3dvs.100) (throughout this document)